pycbc/inference/models/gaussian_noise.py

```diff
@@ -240,10 +242,20 @@ def __init__(self, variable_params, data, waveform_generator,
         # we'll use the first data set for setting values
         d = data.values()[0]
         N = len(d)
-        # figure out the kmin, kmax to use
-        self._f_lower = f_lower
-        kmin, kmax = pyfilter.get_cutoff_indices(f_lower, f_upper, d.delta_f,
-                                                 (N-1)*2)
+        # lower frequency cutoff gets set from config file
+        try:
+            lower_frequency_cutoff = float(lower_frequency_cutoff)
+            self._f_lower = lower_frequency_cutoff
+        except (ValueError, TypeError):
+            if lower_frequency_cutoff is None:
+                raise KeyError("lower frequency cutoff "
+                               "needs to be specified in config (.ini) file")
+            else:
+                raise ValueError("lower frequency cutoff must be convertable "
+                                 "to float but type is "
+                                 "{}".format(type(lower_frequency_cutoff)))
```

Comment:
You don't need this block of code. `low_frequency_cutoff` should already be a float when it's passed to the `__init__` function; if it isn't, that's a bug in the way the class is being initialized. Also, by not making `low_frequency_cutoff` a keyword arg, it will have to be provided.